packages/documentation/docs/controls/buttons/_toggle-button_styleguide.md

```diff
@@ -0,0 +1,28 @@
+Toggle buttons are button elements which allow the user to activate/deactivate a function. Toggle buttons with and without text labels are available. We typically use toggle buttons within button groups when users can chose between more than two options or when two available option don't follow the on/off metaphor. 
```

Comment:
Again button group can be missleading

```suggestion
Toggle buttons are button elements which allow the user to activate/deactivate a function. Toggle buttons with and without text labels are available. We typically use toggle buttons within button groups when users can chose between more than two options or when two available options don't follow the on/off metaphor. 
```

Reply:
Here I actually meant the button group as a component. I tried to make it more clear